pyproject.toml

```diff
@@ -16,8 +16,8 @@ classifiers = [
     "Operating System :: OS Independent"
 ]
 [tool.poetry.dependencies]
-python = "^3.6.9"
-aries-cloudagent = "^0.8.1"
+python = "^3.9"
+aries-cloudagent = ">= 0.9.0, < 1.0.0"

```

Comment:
I am good with this for now. I would move ACA-Py to dev dependencies unless we can ensure that the pre-installed version on the image will NOT be overwritten if it doesn't match the expected version.

Reply:
The plugins in traction all have an exact acapy dependency. I think for the plugin repo we should have exact acapy dependencies and then tag them. If the parent application wanted to upgrade it's acapy it would need to upgrade the plugin to the latest tag as well. The plugins would be upgraded and tested when acapy upgrades happen.

This does seem slightly dangerous if acapy and the parent application was upgraded to 0.11.0 and the plugin wasn't tested with it.